internal/app/pocket/delivery/pocket_router.go

```diff
@@ -0,0 +1,175 @@
+package delivery
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/lucky-pocket/luckyPocket-back/internal/domain"
+	"github.com/lucky-pocket/luckyPocket-back/internal/domain/data/input"
+	"github.com/lucky-pocket/luckyPocket-back/internal/global/auth"
+	"github.com/lucky-pocket/luckyPocket-back/internal/global/error/status"
+)
+
+type PocketQuery struct {
+	Offset int `form:"offset" binding:"number"`
+	Limit  int `form:"limit" binding:"number"`
+}
+
+type PocketRequest struct {
+	ReceiverID uint64 `json:"receiverID" binding:"required"`
+	Coins      int    `json:"coins" binding:"number"`
+	Message    string `json:"message" binding:"required"`
+	IsPublic   bool   `json:"isPublic" binding:"required"`
+}
+
+type Visible struct {
+	Visible bool `json:"visible" binding:"required"`
+}
```

Comment:
```suggestion
type VisibilityRequest struct {
	Visible bool `json:"visible" binding:"required"`
}
```
visibilityRequest도 괜찮을 것 같습니다